volttron/platform/vip/agent/subsystems/pubsub.py

```diff
@@ -101,154 +104,41 @@ def __init__(self, core, rpc_subsys, peerlist_subsys, owner):
         self._peer_subscriptions = {}
         self._my_subscriptions = {}
```

Comment:
As there is only one possible peer now so self._my_subscriptions doesn't need to be a dictionary, I think. Also anywhere that references it should now handle the peer argument correctly by ignoring it.

self._peer_subscriptions can go away as it's no longer used.